rfcs/accepted/0001-cli-atomic-taxonomy.md

```diff
@@ -0,0 +1,106 @@
+- Feature Name: Taxonomy for Atomic in the Coveo CLI
+- Start Date: 2023-03-08
+
+# Summary
+
+[summary]: #summary
+
+Create a one-stop-shop topic for CLI user that uses Atomic:
+
+- `atomic:deploy` alias `ui:deploy`
```

Comment:
If we have to define one of those two as the "main" command and the other as an alias, I would prefer if we said "ui:deploy" is the main command and "atomic:deploy" is the alias, since AFAIK, `ui:deploy` is framework-agnostic.

Reply:
Sorry I now noticed, I meant alias as a verb i.e. to alias something. Will clarify